lib/DocumentRetriever.ts

```diff
@@ -15,8 +15,9 @@ interface DocumentRetrieverTypeInformation {
 	type: DocumentRetrieverTypes;
 	pastTense: string;
 }
+
 // DocumentRetriever is used for both Scan and Query since a lot of the code is shared between the two
-abstract class DocumentRetriever {
+abstract class DocumentRetriever{
```

Comment:
This is one where I'd love it if the linter would fail on this. I don't know why, but this is me being picky. But I do prefer a space before `{`. Just makes it look cleaner IMO.